src/loose/oose/fis/lab/student/manager/Main.java

```diff
@@ -0,0 +1,24 @@
+package loose.oose.fis.lab.student.manager;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class Main extends Application { //
+
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+        Parent root = FXMLLoader.load(getClass().getResource("fxml/view-students.fxml"));
+        primaryStage.setTitle("Student Manager");
```

Comment:
I would like the name for the application to be “SEF - Student Manager”
```suggestion
        primaryStage.setTitle("Student Manager");
```